apps/wakunode2/wakunode2.nim

```diff
@@ -383,6 +385,15 @@ proc setupProtocols(node: WakuNode, conf: WakuNodeConf,
       let pubsubTopics = conf.topics.split(" ")
       await mountRelay(node, pubsubTopics, peerExchangeHandler = peerExchangeHandler)
     except CatchableError:
+
+      # Get this from cli
+      var topicsPublicKeys = initTable[string, SkPublicKey]()
+
+      # Add validation keys to protected topics
+      for topic, publicKey in topicsPublicKeys.pairs:
+        info "routing only signed traffic", topic=topic, publicKey=publicKey
+        node.wakuRelay.addSignedTopicValidator(Pubsubtopic(topic), publicKey)
+    except:
```

Comment:
I'm pretty sure this is a faulty merge conflict resolution. This should not be part of the `except CatchableError` block and the error-less `except` in L396 should be `except CatchableError`.